clients/drcachesim/scheduler/scheduler.h

```diff
@@ -522,6 +522,17 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
          * blocking and trigger a context switch.
          */
         uint64_t blocking_switch_threshold = 100;
+        /**
+         * Controls the amount of time inputs are considered blocked at a syscall whose
+         * latency exceeds #syscall_switch_threshold or #blocking_switch_threshold.  A
+         * "block time factor" is computed from the syscall latency divided by either
+         * #syscall_switch_threshold or #blocking_switch_threshold.  This factor is
```

Comment:
These thresholds are applied in raw2trace whereas the block_time_scale is applied post-raw2trace. If the thresholds change after raw2trace (and therefore the maybe_blocking markers) would that be okay?